frontend/viewer/src/project/ProjectSidebar.svelte

```diff
@@ -23,9 +22,9 @@
 
   function handleProjectSelect(selectedProject: IProjectModel) {
     if (selectedProject.fwdata) {
-      navigate('/fwdata/' + selectedProject.name);
+      navigate('/fwdata/' + selectedProject.code);
```

Comment:
This is a semantic change to the URL. Does the endpoint now expect a code instead of a (human-readable) name? I don't see a corresponding endpoint change in this PR, but maybe I'm missing something and it's already there.

Reply:
fwdata projects don't have a name **and** a code. They just have one.
So, typically for fwdata projects the `name` and `code` properties are the same: the file-system name of the project.
In that case it doesn't matter if we reference `name` or `code` here.

But if there are crdt **and** fwdata copies of the same project (i.e. crdt project code == fwdata file-system name, which is often the project code), those are both represented by the same `IProjectModal` instance and then the `name` property is the Lexbox/crdt project name, which doesn't mean anything in the context of an fwdata project.